src/shell/lisa_shell

```diff
@@ -74,12 +74,12 @@ echo -ne "$LISASHELL_DEFAULT"
 ################################################################################
 
 function _lisa-update-usage {
-    echo "Usage: lisa-update (CMD)"
+	echo "Usage: lisa-update (CMD)"
```

Comment:
This means you are not using VIM? :)

... since this conflicts with the modeline at the end :/

Reply:
The modeline at the end is:
`# vim: set tabstop=4:`
From Vim doc: "Number of spaces that a Tab in the file counts for."

This does not tell vim that you need to use tabs or spaces, and since you introduced this mixture in 33108ec6, you probably want to review these bits of configuration in your setup :-)
Currently, there is no way to avoid review comments: either you use spaces and somebody will notice a nearby line that uses tabs, or you use tabs and somebody will notice a nearby line uses spaces, so either we don't care at all or we choose either tabs or spaces.

Reply:
Right... I think that's my local setup where I also set expand tab... but... anyway... if that's a fix across all the file I'm still ok with it... just keep it on a dedicated patch (which you already do) and check that what you get is still a reasonable indentation... for example lines 253-255 in fb164a7 seems to be a bit ugly

Reply:
Right, I missed that one and line 340